tests/unit_tests/external_interaction/ispyb/test_gridscan_ispyb_store_2d.py

```diff
@@ -138,35 +128,57 @@ def dummy_collection_group_info(dummy_params):
     "hyperion.external_interaction.callbacks.common.ispyb_mapping.get_current_time_string",
     new=MagicMock(return_value=EXPECTED_START_TIME),
 )
-def scan_data_info_for_begin(dummy_params):
-    grid_scan_info = GridScanInfo(
-        dummy_params.hyperion_params.ispyb_params.upper_left,
-        dummy_params.experiment_params.y_steps,
-        dummy_params.experiment_params.y_step_size,
-    )
-    info = ScanDataInfo(
-        data_collection_info=populate_remaining_data_collection_info(
-            lambda: construct_comment_for_gridscan(
-                dummy_params, dummy_params.hyperion_params.ispyb_params, grid_scan_info
-            ),
-            None,
-            populate_xy_data_collection_info(
-                grid_scan_info,
-                dummy_params,
-                dummy_params.hyperion_params.ispyb_params,
-                dummy_params.hyperion_params.detector_params,
-            ),
-            dummy_params.hyperion_params.detector_params,
-            dummy_params.hyperion_params.ispyb_params,
+def scan_data_info_for_begin():
```

Comment:
Could: I think it would be nicer if we could combine some of the common bits of these test helper functions that make `ScanDataInfo` and `DataCollectionInfo` as they're very verbose but I appreciate this makes it more explicit so up to you.

Reply:
Thinking maybe I should remove this file if we don't support 2D gridscans!